src/main/java/seedu/address/model/person/Session.java

```diff
@@ -54,7 +54,8 @@ public static boolean isValidSession(String test) {
         }
         String day = test.substring(0, 3);
         String hour = test.substring(4);
-        return day.matches(VALIDATION_REGEX_DAY) && hour.matches(VALIDATION_REGEX_TIME) && isValidDay(day);
+        return day.matches(VALIDATION_REGEX_DAY) && hour.matches(VALIDATION_REGEX_TIME) && isValidDay(day)
+                && isValidHour(hour);
```

Comment:
wait just kidding maybe this should be time instead of hour cos ure testing minute also

Reply:
but ya other than that lgtm!!

Reply:
hmm okkiee thank youu